yandex/data_source_yandex_vpc_security_group.go

```diff
@@ -0,0 +1,143 @@
+package yandex
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+
+	"github.com/yandex-cloud/go-genproto/yandex/cloud/vpc/v1"
+)
+
+func dataSourceYandexVPCSecurityGroup() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceYandexVPCSecurityGroupRead,
+		Schema: map[string]*schema.Schema{
```

Comment:
There is no field `security_group_id` in Schema which should used to lookup Security Group.